minidns-core/src/main/java/org/minidns/record/SVCB.java

```diff
@@ -0,0 +1,102 @@
+package org.minidns.record;
+
+import org.minidns.dnsname.DnsName;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * SVCB Record Type (Service binding)
+ *
+ * https://tools.ietf.org/html/draft-ietf-dnsop-svcb-httpssvc-01
```

Comment:
This link, and all other links in javadoc in this file, should ideally be using `@see`. For example in this case

```
* @see <a href="https://tools.ietf.org/html/draft-ietf-dnsop-svcb-https-01">draft-ietf-dnsop-svcb-https-01: Service binding and parameter specification via the DNS (DNS SVCB and HTTPS RRs)</a>
```

Reply:
Sure, I'm going to add it.